owners/index.js

```diff
@@ -33,84 +32,84 @@ const INFO_SERVER_PORT = Number(process.env.INFO_SERVER_PORT || 8081);
 module.exports = app => {
   const localRepo = new LocalRepository(process.env.GITHUB_REPO_DIR);
   const ownersBot = new OwnersBot(localRepo);
-  const github = new GitHub(
+  const sharedGithub = new GitHub(
     new Octokit({auth: `token ${GITHUB_ACCESS_TOKEN}`}),
     GITHUB_REPO_OWNER,
     GITHUB_REPO_NAME,
     app.log
   );
-  const teamsInitialized = ownersBot.initTeams(github);
+  ownersBot.initTeams(sharedGithub);
+
+  /**
+   * Listen for webhooks and provide handlers with a GitHub interface and the
+   * event payload.
+   *
+   * @param {!string|string[]} events event or list of events to listen to.
+   * @param {!function} cb callback function.
+   */
+  function listen(events, cb) {
+    app.on(events, async context => {
+      let github;
+      try {
+        github = GitHub.fromContext(context);
+      } catch (e) {
+        // Some webhooks do not provide a GitHub instance in their context.
```

Comment:
😬

Reply:
It's because some are based off of user actions (like creating a PR) while others are "GitHub" actions (like altering team membership or a PR being merged). It's weird. :man_shrugging: